doc/Developing.md

```diff
@@ -86,32 +86,45 @@ Some additional tools and libraries are needed depending on your platform:
 
 To build and run the browser example:
 
-    git clone https://github.com/eclipse-theia/theia \
+```sh
+git clone https://github.com/eclipse-theia/theia \
     && cd theia \
     && yarn \
-    && cd examples/browser \
-    && yarn run start
+    && yarn browser build \
+    && yarn browser start
```

Comment:
Given that it does not automatically happen anymore, do we need to mention to download plugins?